test/operator/test_operator.py

```diff
@@ -579,6 +579,7 @@ def test_jax_operator_to_jax_operator(op):
     assert op == op.to_jax_operator()
 
 
+@common.skipif_sharding
```

Comment:
@gcarleo @PhilipVinc unrelated to this bug but i think this should be applied as well.